pkg/apis/tensorflow/v1/types.go

```diff
@@ -56,6 +56,11 @@ type TFJobSpec struct {
 	// +optional
 	BackoffLimit *int32 `json:"backoffLimit,omitempty"`
 
+	// SuccessPolicy defines the policy to mark the TFJob as succeeded.
+	// Default to "", using the default rules.
+	// +optional
+	SuccessPolicy *SuccessPolicy `json:"successPolicy,omitempty"`
```

Comment:
can limit the successPolicy value by adding like: // +kubebuilder:validation:Enum=Lion;Wolf;Dragon?